src/handler/websocket.rs

```diff
@@ -80,11 +80,5 @@ async fn setup_sender(state: &super::super::AppState, group_id: &str) -> broadca
     let mut group_list = state.group_list.lock().await;
     let group_id = group_id.to_string();
 
-    match group_list.get_mut(&group_id) {
-        None => {
-            group_list.insert(group_id, tx.clone());
-            tx
-        }
-        Some(group) => group.clone(),
-    }
+    group_list.entry(group_id).or_insert(tx).clone()
```

Comment:
Do you prefer [`or_insert_with`](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html#method.or_insert_with) ?

Reply:
```
    group_list.entry(group_id).or_insert_with(|| {
        let (tx, _rx) = broadcast::channel(100);
        tx
    }).clone()
```

I thought that writing it this way would be a saving memory in temporary.

Reply:
Good! I use it.